src/parser/parsed_data/attach_info.cpp

```diff
@@ -15,6 +14,8 @@ StorageOptions AttachInfo::GetStorageOptions() const {
 			storage_options.block_alloc_size = entry.second.GetValue<uint64_t>();
 		} else if (entry.first == "row_group_size") {
 			storage_options.row_group_size = entry.second.GetValue<uint64_t>();
+		} else if (entry.first == "compatibility_version") {
+			storage_options.compatibility_version = entry.second.GetValue<string>();
```

Comment:
Do we want to validate the option here and store the internal version number instead of a string in the storage options?